src/evict/evict_lru.c

```diff
@@ -2383,6 +2383,8 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
         if (!F_ISSET(conn, WT_CONN_RECOVERING) && __wt_cache_stuck(session)) {
             ret = __wt_txn_is_blocking(session);
             if (ret == WT_ROLLBACK) {
+                __wt_verbose_debug(
+                  session, WT_VERB_TRANSACTION, "Rollback reason: %s", "Cache full");
```

Comment:
np: Do we need to string format this? i.e. can it just be `__wt_verbose_debug(session, WT_VERB_TRANSACTION, "Rollback reason: Cache full");`

Reply:
Nice, I fixed the others and forgot this one.

Reply:
In fact, we can't with our definition of `__wt_verbose_debug`:
```
#define __wt_verbose_debug(session, category, fmt, ...) \
    __wt_verbose_level(session, category, WT_VERBOSE_DEBUG, fmt, __VA_ARGS__)
```
We get the following compilation error:
```
../src/evict/evict_lru.c: In function '__wt_cache_eviction_worker':
../src/include/verbose.h:103:77: error: expected expression before ')' token
             __wt_verbose_worker(session, "[" #category "] " fmt, __VA_ARGS__); \
                                                                             ^
../src/include/verbose.h:132:5: note: in expansion of macro '__wt_verbose_level'
     __wt_verbose_level(session, category, WT_VERBOSE_DEBUG, fmt, __VA_ARGS__)
     ^~~~~~~~~~~~~~~~~~
../src/evict/evict_lru.c:2386:17: note: in expansion of macro '__wt_verbose_debug'
                 __wt_verbose_debug(
```

Reply:
We could rework our definition of `__wt_verbose_debug` and check if `__VA_ARGS__` is empty and avoid passing it to `__wt_verbose_level`.

Reply:
Ah right, got it. I missed the detail in the original comment:
```
* The macro must take a format string and at least one additional argument, there's no portable way to remove the comma before an empty __VA_ARGS__ value.
```
Researching some work around, they all seem a bit hacky. Happy to leave it as it is, thanks for testing this!